.github/workflows/publish-release.yml

```
@@ -43,4 +43,6 @@ jobs:
       - name: Publish to GitHub Pages
         uses: MetaMask/action-publish-gh-pages@v2
         with:
-          source-directory: package/site/build
+          source-directory: packages/site/dist
```

Gatsby creates the production build under `public`.

```suggestion
          source-directory: packages/site/public
```